### ISSUE TITLE
add  upgradehacknet option in hax,js

### ISSUE THREAD

**Issue by akara0775** [ChaelCodes/bitburner-scripts#7]
this is not really an issue per se, just tiny improvement

i manually add this in hax.js
```
case upgradeHacknet:
  ns.run(`/${getFolder()}/buyHacknet.js`, 1, 'upgradeNodes');
  break;
```

**Comment by ChaelCodes** [comment]
I like this! I'm always a fan of adding new commands. Are you familiar with the PR creation process? Would you like to add it to the repo?

**Comment by akara0775** [comment]
im not familiar with those process but interested in learning

**Comment by ChaelCodes** [comment]
Here's an article on how to do it. https://opensource.com/article/19/7/create-pull-request-github

After you fork the repository, you don't actually have to clone it to your local machine. You can edit the file inside GitHub. As long as you commit the changes to a new branch, and make a PR, I'll be able to add it to this repo.

Link me to your PR once you've made it! 👍

**Comment by akara0775** [comment]
fix #8

**Comment by ChaelCodes** [comment]
@akara0775 Thanks! The PR looks perfect. I'll be able to test and merge it after the holidays.

Thank you!
